scripts/version.py

```diff
@@ -84,7 +85,7 @@ def main() -> None:
             version = normalize_version(version)
         print(version)
     elif args.command == "set":
-        set_version(args.toml_file, args.key, args.version)
+        update_versions(args.entries, args.version)
```

Comment:
May want to call `update` `set`, but seems fine.

Reply:
I'm working on another branch now so will just add as is, but can change later on